src/main/java/org/sefglobal/scholarx/service/ProgramService.java

```diff
@@ -116,6 +120,44 @@ public Program updateProgram(long id, Program program) throws ResourceNotFoundEx
      */
     public Program updateState(long id) throws ResourceNotFoundException {
         Optional<Program> program = programRepository.findById(id);
+        List<Mentor> mentors = mentorRepository.findAllByProgramId(id);
+        List<Mentor> approvedMentors = mentorRepository.findAllByProgramIdAndStateIn(id, Collections.singletonList(enrolmentState.APPROVED));
+        List<Mentee> mentees = menteeRepository.findAllByProgramId(id);
+
+        if (program.get().getState().next().toString().equals("MENTEE_APPLICATION")) {
```

Comment:
Instead of equaling to custom `Strings` use the existing enums for program states.
Follow this pattern for the rest of the code.